modules/developers-guide/examples/go/go-auth-secure.go

```diff
@@ -0,0 +1,16 @@
+grpcEndpoint := "localhost:8090"
+authEndpoint := "localhost:8081"
+username := "cassandra"
+passwd := "cassandra"
+
+config := &tls.Config{
+ 	InsecureSkipVerify: false,
```

Comment:
`false` is the default so this doesn't need to be set. Here's one recommended way of specifying TLS in the go grpc client: https://github.com/grpc/grpc-go/blob/master/Documentation/grpc-auth-support.md#enabling-tls-on-a-grpc-client

Reply:
I'll set this as an item to update, after I have time to investigate. Thank you for pointing it out, Michael.